code/game/machinery/machinery_power.dm

```diff
@@ -92,18 +92,18 @@
 	var/old_loc = loc
 	if((. = ..()))
 		update_power_on_move(src, old_loc, loc)
-		if(ismovable(loc)) // Register for recursive movement (if the thing we're inside moves)
+		if(ismovableatom(loc)) // Register for recursive movement (if the thing we're inside moves)
```

Comment:
Instead of these changes we can just add the line `#define ismovable(A) (istype(A, /atom/movable))`  to ___513_compatibility.dm_